xlators/mgmt/glusterd/src/glusterd-syncop.c

```diff
@@ -224,7 +224,6 @@ glusterd_syncop_aggr_rsp_dict(glusterd_op_t op, dict_t *aggr, dict_t *rsp)
     xlator_t *this = NULL;
 
     this = THIS;
```

Comment:
So 'this' is used exactly once in this function - in an error log. Do we really need it at all?
Instead of:
```
gf_msg(this->name, GF_LOG_ERROR, 0,
                       GD_MSG_BRICK_MOUNDIRS_AGGR_FAIL,
                       "Failed to "
                       "aggregate brick mount dirs");
```
Let's do:
```
gf_msg(THIS->name, GF_LOG_ERROR, 0,
                       GD_MSG_BRICK_MOUNDIRS_AGGR_FAIL,
                       "Failed to "
                       "aggregate brick mount dirs");
```

And get rid of the xlator_t* parameter and the assignment.

Reply:
Ack..I'll go through the management layer one more time to see if there are such instances which I've missed.